pkg/asset/agent/image/ignition.go

```diff
@@ -48,6 +51,7 @@ type agentTemplateData struct {
 	ReleaseImageMirror    string
 	MirrorRegistriesMount string
 	CaBundleMount         string
+	InfraEnvID            strfmt.UUID
```

Comment:
Just make this a string, then we won't have to modify go.mod.
There's no actual advantage to using a strfmt.UUID (which is just a string), given that we are casting it to that type, and then using it as a string so there is no validation at any point.

Reply:
Done